tests/plugins/wysiwygarea/manual/detachobserver.html

```diff
@@ -0,0 +1,33 @@
+<div id="editor">
+	<p>Lorem ipsum dolor sit amet.</p>
+</div>
+<button id="triggerButton" disabled>Trigger test</button>
+
+<script>
+	( function() {
+		if ( bender.tools.env.mobile ) {
```

Comment:
`MutationObserver`is not supported in IE below 11. We should add it to this condition.

```suggestion
		if ( bender.tools.env.mobile && CKEDITOR.env.ie && CKEDITOR.env.version < 11 ) {
```